src/core/constants.js

```diff
@@ -299,3 +299,6 @@ export const RATINGS_MODERATE = 'Ratings:Moderate';
 export const USERS_EDIT = 'Users:Edit';
 // Super powers. It means absolutely all permissions.
 export const ALL_SUPER_POWERS = '*:*';
+
```

Comment:
Should I add a comment here like `Left-to-right and right-to-left variables` .

Reply:
nope